doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -101,7 +101,31 @@ This profile is the global settings for the whole NFVI, which means there should
 | Crypto Acceleration Interface | N  | Y | Y |
 
 <a name="3.4"></a>
-## 3.4 NFVI Required State
+## 3.4 NFVI Hardware Profile
+[RM1: 5.4 NFVI HW profiles features and requirements](../../../ref_model/chapters/chapter05.md#5.4) defines the NFVI hardware layer profiles. It is difficult to require all labs to follow this hardware profile and prepare seperate hardware sets for different profiles. However, when reference implementations and the follow up testing and verification are conducted, the hardware profile need to be clearly described. This is especially important for performance testing and verification. 
```

Comment:
Re: "It is difficult to require all labs to follow this hardware profile and prepare seperate hardware sets for different profiles."

Should this be reworded so that it is clear why. For example, "The labs are typically provisioned with the minimal required hardware and thus it is difficult to partition the available hardware to provision/configure multiple NFVI profiles."